src/main/java/com/adyen/enums/Region.java

```diff
@@ -1,8 +0,0 @@
-package com.adyen.enums;
-
-public enum Region {
```

Comment:
Can we keep the original `Region` enum class, otherwise this change will be breaking